ui.frontend/src/view/FormField.js

```diff
@@ -77,10 +77,14 @@ class FormField {
     setActive() {
         if (!this.isActive()) {
             this.element.setAttribute(Constants.DATA_ATTRIBUTE_ACTIVE, true);
+            if(this.parentView && this.parentView._model.activeChild !== this._model) { 
+                this.parentView._model.activeChild = this._model; // updating the activeChild of the model when a field is focused in view
```

Comment:
this code should go in af2 web runtime and should be navigated by model and not view.

Reply:
But how will we handle the scenario where user clicks on a field and we have to set the Active child in model??

Reply:
Currently model is navigating view that's working but this is for the other way round.